content/docs/user-guide/interactivity/physics/nvidia-physx/configuring/configuration-collision-layers.md

```diff
@@ -1,40 +1,43 @@
 ---
-description: ' Create collision layers for the PhysX system in Open 3D Engine. '
-title: Collision Layers
-weight: 200
+linkTitle: Collision Layers
+title: PhysX Collision Layers
+description: ' Create collision layers for the PhysX system in Open 3D Engine (O3DE). '
+weight: 300
+toc: true
 ---
 
-Use collision layers to group objects of the same type. Use collision groups to define what the collision layers collide with. You can have a maximum of 64 layers per game.
+With collision layers, you can place related PhysX entities into categories. The following list demonstrates some example PhysX collision layers:
 
-Common types of layers are objects like the following:
-+ Terrain
-+ Static objects
-+ Players
-+ Enemies
-+ Projectiles
+* **Terrain** - Terrain, flooring, and any entities that player entities can traverse.
+* **Static objects** - Entities that have colliders, but no animation or rigid body components, such as large rocks, tree trunks, and walls.
+* **Players** - All player controlled entities.
+* **Enemies** - Entities that move via script or AI that can deal damage to, and receive damage from, player controlled entities.
+* **Projectiles** - Entities that can deal damage.
 
-The layers that you define are specific to your game. When a collider is created, it is assigned a collision layer with index \[0\] called `Default`. This can't be changed.
+A project can have up to 64 PhysX collision layers. The layers that you define are specific to your project. When a **PhysX Collider** component is added to an entity, it is assigned a collision layer with index **\[0\]** called `Default`. This can't be changed. You can assign each collider component to one layer. An entity can have multiple collider components assigned to different layers.
```

Comment:
The sentence `This can't be changed.` it's not clear to me what can't be changed. Does this mean that the name `Default` cannot be changed? It reads like the collision layer must have `Default` and cannot be changed.